Source/Core/VideoCommon/ShaderCache.cpp

```diff
@@ -1339,25 +1339,33 @@ const AbstractPipeline* ShaderCache::GetTextureReinterpretPipeline(TextureFormat
   return iiter.first->second.get();
 }
 
-const AbstractShader* ShaderCache::GetTextureDecodingShader(TextureFormat format,
-                                                            TLUTFormat palette_format)
+const AbstractShader*
+ShaderCache::GetTextureDecodingShader(TextureFormat format,
```

Comment:
It's rather minor but my suggestion is rather than sending down `DecodingShaderInfo` we use `std::optional<TLUTFormat> palette_format` instead.  It's a little clearer and if we ever refactor to move `DecodingShaderInfo` somewhere else, we wouldn't need to include it in `ShaderCache`  (this class already uses `optional`)

Reply:
Like this?

Reply:
Yes, thank you very much!